cursor_testowanie_aplikacji_z_u_yciem_p.md

```
@@ -0,0 +1,618 @@
+# Testowanie aplikacji z użyciem Playwright lub Puppeteer
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)